### ISSUE TITLE
shapely import disables logging

### ISSUE THREAD

**Issue by nheeren** [Toblerity/Shapely#315]
Rather a minor thing, but this is a little inconvenience for my code at least
http://stackoverflow.com/q/32245251/2075003
Maybe you want to look into this.

**Comment by mwtoews** [comment]
This should be fixed by #313

**Comment by nheeren** [comment]
Thanks & sorry I missed that!

**Comment by sgillies** [comment]
@nheeren Shapely 1.5.12 is on PyPI now.

**Comment by nheeren** [comment]
Totally works! Great!

**Comment by nheeren** [comment]
Sorry to bother again... So 1.5.12 fixed the bug. But I noticed something else: 
When I initialise ´basicConfig´ with `level=DEBUG` before importing e.g. `from shapely.geometry import Point`, the following messages are logged: 

    DEBUG (libgeos.py get_geos_config): geos-config --clibs: '-L/usr/local/Cellar/geos/3.4.2/lib -lgeos_c'
    DEBUG (libgeos.py <module>): found GEOS C library using geos-config

I am not sure if this intentional, but none of the other packages I usually import does that. In my case I imported a settings file before importing shapely. Again, only a minor thing. Cheers, Niko.
